types/highland/index.d.ts

```diff
@@ -1009,6 +1033,8 @@ declare namespace Highland {
 		 */
 		latest(): Stream<R>;
 	}
+
+	type MappingHint = number | string[] | Function;
```

Comment:
@DanielRosenwasser was is the difference between `Function` and `((...args: any[]) => any)`?